src/osvr/Server/ConfigFilePaths.h.cmake_in

```diff
@@ -0,0 +1,57 @@
+/** @file
+    @brief Auto-generated configuration header - do not edit!
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_ConfigFilePaths_h_GUID_241E9C9C_0E0E_46B0_9DED_8F8059306192
+#define INCLUDED_ConfigFilePaths_h_GUID_241E9C9C_0E0E_46B0_9DED_8F8059306192
+
+#include <osvr/Util/PlatformConfig.h>
```

Comment:
FYI, in case you hadn't seen it already, there's a boilerplate generator that will spit stuff out that looks like this (including the little headings for categories of include files) available here: http://tools.getosvr.org/boilerplate/  Please leave those headings in place, if possible, to help keep consistent style and organization.

A gist with some slick Windows-only integrations is here (puts it in your send-to menu), making a shell script variant (which is what this originally was before a PHP page) should be pretty easy: https://gist.github.com/rpavlik/fbdf1347e52b64b833c5